core/src/main/java/tech/pegasys/eth2signer/core/Runner.java

```diff
@@ -73,6 +78,19 @@ public void run() {
         .failureHandler(new LogErrorHandler())
         .handler(routingContext -> routingContext.response().end("OK"));
 
+    final SigningMetadataTomlConfigLoader configLoader =
+        new SigningMetadataTomlConfigLoader(config.getKeyConfigPath());
+    final ArtefactSignerProvider signerProvider = new MultiKeyArtefactSignerProvider(configLoader);
+
+    final SigningRequestHandler handler = new SigningRequestHandler(signerProvider);
+
+    router
+        .routeWithRegex(HttpMethod.POST, "/sign/" + "(attestation|block)")
```

Comment:
signer instead of sign? most of the eth2 apis seem to have a noun for the first part of the of the path.

Reply:
done